README.md

```
@@ -71,9 +71,9 @@ git merge main
   </tr>
   <tr>
     <td align="center" width="150">
-      <a href="#">
-        <img src="/profile-pictures/placeholder.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
-        <b>Name</b><br/><br/>
+      <a href="http://www.linkedin.com/in/chloexthompson">
+        <img src="/profile-pictures/chloe-thompson.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)